src/ua/hillel/Array.java

```diff
@@ -0,0 +1,36 @@
+package ua.hillel;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Array {
```

Comment:
The name of a class should tell a developer what it can do. It's not really about an array. Would be better to rename it to `WordGuessingGame`